ansible/README.md

```diff
@@ -97,6 +97,20 @@ Yes, in order to access the package repositories (we will perform either `yum in
 
 4) The Ansible playbook will download and install any dependencies needed to build OpenJDK
 
+## How do I run the playbook on a Windows Machine?
+
+As Ansible can't run on Windows, it has to be run on a seperate system and then pointed at a Windows machine (such as a VM). To test the playbook, a vagrant VM can be used.
```

Comment:
Thanks for the update, but I'll still bite here :-) - https://www.ansible.com/integrations/infrastructure/windows why can't we run on windows directly?

Reply:
I was going off what the FAQs say in the Ansible documentation:
https://docs.ansible.com/ansible/latest/user_guide/windows_faq.html

Also doing it this way seemed easier than dealing with the problems found in issue #791

Reply:
@karianna That page is talking about ansible's Windows modules that you can use when remotely connecting in, not executing the playbooks natively on the host. I believe it should be doable but it's a bit of a pig to set up. We have https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/791 to cover anything we can do to make this easier in the future, but at the moment we don't easily support it or test it.

Reply:
Got it thanks both.